packages/app/obojobo-repository/shared/actions/dashboard-actions.js

```diff
@@ -8,6 +8,8 @@ dayjs.extend(advancedFormat)
 
 const JSON_MIME_TYPE = 'application/json'
 const XML_MIME_TYPE = 'application/xml'
+const XML_MIME_TYPE1 = 'text/xml'
```

Comment:
I might prefer naming these `XML_MIME_TYPE_APPLICATION` and `XML_MIME_TYPE_TEXT` to be more descriptive.